### ISSUE TITLE
Configurator wiki doesn't explain how to use configurator

### ISSUE THREAD

**Issue by Tejeev** [kiibohd/KiiConf#25]
Link from configurator goes to code page
what does /f/1 do?
How do you import a configuration you downloaded from the configurator in the past?  It's not like it gave me a  json but it sure asks for one... Don't know where to go from there.
Is there a primer on how the layer locks work anywhere?  I've been looking but I've yet to find anything yet.  If it's there it should be linked to prominently

**Comment by haata** [comment]
If the configuration isn't *too* old, the .zip file would have come with a .json file. You can copy the contents of this file to import the configuration. If it's really old, you'll have to make a new configuration...sorry about that.

There are 3 types of function modifier keys.
Shift, i.e. /f/1
Lock
Latch

Shift, temporarily applies that layer to the keyboard while held.
Lock, applies that layer until a lock key is pressed again (or you press shift or latch to temporarily untoggle)
Latch, applies that layer until the next key is pressed (think sticky keys)

Yeah, I think we need some better documentation available from the configurator page (you're not the first person to ask about it).
I'll leave this bug open until we fix this.

**Comment by Tejeev** [comment]
Thanks haata!
We talk occasionally on IRC so I'm glad to see you respond.  :p
I just used the configurator at the .club site and it did not come with json :/  Should I just run one of my own on my own computer?

**Comment by haata** [comment]
Sorry for the delay. Can you try using the web configurator again and send
me the filename of the .zip file? (I can pull the cache from the server to
see what sort of issues there are).

**Comment by Tejeev** [comment]
yeah, Ill try to take care of it this weekend.  Just got back after traveling for two weeks so still settling in.

**Comment by nanliu** [comment]
Also is there an example for how to implement double quotes?

**Comment by haata** [comment]
@nanliu using KLL?

One way would be using the ascii convenience macros to do Shift + single quote
`U"A" : '"';`
which is the same as
`U"A" : U"Shift" + U"Quote";`

**Comment by nanliu** [comment]
@haata I was trying to specify double quotes via the web configurator. I tried tweaking the json config using "\"", but that didn't work. I'll try the KLL compiler.

**Comment by jgillman** [comment]
Apologies in advance for the slew of questions, but I'm also confused by the title problem @Tejeev was having. I understand the basics of how the Configurator works (setting keys on different layers) but I don't understand what a lot of the shorthand means. Some kind of guide or legend would certainly help the people (like myself) who aren't familiar with layers or it's related lingo.

IDK what any of these mean:

- `NEXT-ƒ`
- Really anything with the fancy `ƒ` in it
- `LATCH-`
- `LOCK-`

Also what is the "simple macros" section for? Are those literally keys for `@`, `*`, and so on? Or do they have special meanings? I'm using a Kinesis at the moment and on that keyboard a "macro" is a series of keys that is "played" when pressing another key. E.g. when I hit a single key it can act like I've just hit several keys.

What's the difference between `NONE` and `CLR-KEY`?

I would be happy to put together a wiki explaining all of that for people like myself once I understand what's what! :smile:

**Comment by robertdale** [comment]
`LATCH` and `LOCK` are already explained above.

`f`# is a momentary switch to layer # 

`NEXT-f` would switch to the next layer

`CLR-KEY` clears the mapping to that key so that it falls through the layers through base until it finds a mapping (or not). 

`NONE` makes the key literally do nothing regardless of how the key is mapped on lower layers or base.

**Comment by skrobul** [comment]
came here from google search looking for answers. does anyone know answer to @jgillman last question about simple macros?

**Comment by alecloudenback** [comment]
I'm uncertain how to configure/use the subscript/superscript keys. For example, in the default layout I'm referring to: 
![capture](https://user-images.githubusercontent.com/711879/30676656-c84cf742-9e4c-11e7-8c64-5ce3c44895e5.PNG)

1. How do you use these keys?
2. How do you add them to keys on the configurator?

And a likely unrelated question:

3. Is it possible to use dual-function/longpress like on the Ergodox EZ? Based on [an issue](https://github.com/kiibohd/controller/issues/2) over at the kiibohd/controller repo it seems like a limitation of the KLL format that the Infinity uses?

If someone could clarify this I would be willing to touch it up and submit a pull request clarifying these in the

**Comment by jbondeson** [comment]
@alecloudenback
1) the keys you see with sub and super scripts are actually different function layers. The colors correspond to the color you see in "Layer X" in the tabs above. The checkboxes enable and disable the visibility of the layers on the UI (it has no impact on what is actually generated, only on the UI).

2) You change them by selecting the different layer and then for base alpha/numerics press them on the keyboard, and for others the keys are shown below.

3) Correct, that is currently a limitation, though @haata has been working on a solution to that in KLL.
